spring-integration-redis/src/test/java/org/springframework/integration/redis/util/RedisLockRegistryTests.java

```diff
@@ -692,6 +692,63 @@ public void multiRedisLockRegistryTest() throws InterruptedException, ExecutionE
 	}
 
 
+	@Test
+	@RedisAvailable
+	public void earlyWakeUpTest() throws InterruptedException {
+		final int THREAD_CNT = 3;
```

Comment:
We use only 2 threads in the `ExecutorService`.
Would you mind to revise, please?